src/nl/hannahsten/texifyidea/util/Magic.kt

```diff
@@ -304,7 +304,8 @@ object Magic {
             "\\" + SECTION.command to "sec",
             "\\" + SUBSECTION.command to "subsec",
             "\\" + SUBSUBSECTION.command to "subsubsec",
-            "\\" + ITEM.command to "itm"
+            "\\" + ITEM.command to "itm",
+            "\\lstinputlisting" to "lst"
```

Comment:
I'm trying to use the commands from LatexRegularCommand everywhere, instead of just putting the command name directly (like with the other things in this map). I'm not 100% sure about it yet, but at the moment the idea is that LatexRegularCommand contains those (and only those) commands for which TeXiFy has special behaviour, and then you can easily look up where a command is used within TeXiFy by find usages instead of text search (which doesn't work for all commands).
 (I know that currently LatexRegularCommand contains too much commands, but I'm planning to extract those automatically from LaTeX package source files so they are just there until then for the autocomplete)

Reply:
Makes sense, I will fix it.